Plots/Contours/NCL_h_long_5.py

```diff
@@ -0,0 +1,129 @@
+ds = xr.open_dataset(gdf.get("netcdf_files/uvt.nc"), cache=False)
+
+# Choose the specific data to use
+U = ds.U.isel(time=0)
+U = U.sel(lat=-16, method="nearest")
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(8, 8))
+ax = plt.axes()
+
+# Set y-axis to have log-scale
+plt.yscale('log')
+
+# Specify which contours should be drawn
+levels = np.linspace(-55, 55, 23)
+
+# Plot contour lines
+lines = U.plot.contour(ax=ax,
+                       levels=levels,
+                       colors='black',
+                       linewidths=0.5,
+                       linestyles='solid',
+                       add_labels=False)
+
+# Label contour lavels at -10, 0, and 10 and set their backgrounds to be white
+ax.clabel(lines, fmt='%d', levels=[-10, 0, 10])
+[
+    txt.set_bbox(dict(facecolor='white', edgecolor='none', pad=1))
+    for txt in lines.labelTexts
+]
+
+# Plot filled contours
+colors = U.plot.contourf(ax=ax,
+                         levels=levels,
+                         cmap=gvcmap.BlWhRe,
+                         add_labels=False,
+                         add_colorbar=False)
+# Add colorbar
+plt.colorbar(colors,
+             ax=ax,
+             orientation='horizontal',
+             ticks=levels[1::2],
+             drawedges=True,
+             aspect=12,
+             shrink=0.65,
+             pad=0.1)
+
+# Use geocat.viz.util convenience function to set axes tick values
+# Set y-lim inorder for y-axis to have descending values
+gvutil.set_axes_limits_and_ticks(
+    ax,
+    xticks=np.linspace(-180, 178, 7),
+    xticklabels=['180', '120W', '60W', '0', '60E', "120E", ""],
+    ylim=ax.get_ylim()[::-1],
+    yticks=U["lev"])
+
+# Change formatter or else tick values will be in exponential form
+ax.yaxis.set_major_formatter(ScalarFormatter())
+
+# Use geocat.viz.util convenience function to add major tick lines with no
+# minor ticks on lefthand side y axis and some minor ticks on the x axis
+gvutil.add_major_minor_ticks(ax=ax,
+                             x_minor_per_major=2,
+                             y_minor_per_major=1,
+                             labelsize=12)
+
+# Use geocat.viz.util convenience function to add titles and the pressure label
+gvutil.set_titles_and_labels(ax,
+                             maintitle="January 1988",
+                             maintitlefontsize=20,
+                             lefttitle=U.long_name,
+                             lefttitlefontsize=14,
+                             righttitle=U.units,
+                             righttitlefontsize=14,
+                             ylabel=U.lev.long_name,
+                             labelfontsize=16)
+
+# Create second y-axis to show geo-potential height.
+# Currently we're using arbitrary values for height as we haven't figured out
+# how to make this work properly yet
```

Comment:
I would consider making this either a block comment or formatting it to be a text block in the documentation build